app/src/test/java/com/github/multimatum_team/multimatum/util/MockGroupRepository.kt

```diff
@@ -56,6 +56,14 @@ class MockGroupRepository(initialContents: List<UserGroup>) : GroupRepository()
         groups.remove(id)
     }
 
+    override suspend fun rename(id: GroupID, newName: String) {
+        groups[id] = groups[id]!!.copy(name = newName)
+    }
+
+    override suspend fun invite(id: GroupID, email: String) {
+        throw NotImplementedError("MockGroupViewModel.invite is not implemented")
```

Comment:
Here too, the exception time is confusing in my opinion

Reply:
I think the best option is to throw `UnsupportedOperationException` in the `MockGroupRepository` and `NotImplementedError` in the `FirebaseGroupRepository`.